src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -33,15 +34,28 @@ export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
         placeholder: 'Enter your reason',
         value: reason,
         onChange: setReason
-      })
+      }),
+      div([
+        h(Checkbox, {
+          id: 'send-copy_link',
+          checked: sendCopy,
+          onChange: () => setSendCopy(!sendCopy)
+        }),
+        label({ htmlFor: 'send-copy_link', style: { marginLeft: 5 } }, ['Also send me a copy'])
+      ])
```

Comment:
We've got `IdContainer` for this sort of thing; that said, would `LabeledCheckbox` do what you need?

Reply:
Thanks for letting me know about this obj! 👍